django/db/models/expressions.py

```diff
@@ -1295,6 +1295,52 @@ def get_db_converters(self, connection):
         ) + self.target.get_db_converters(connection)
 
 
+class ColPairs(Expression):
+    def __init__(self, alias, targets, sources, output_field):
+        super().__init__(output_field=output_field)
+        self.alias, self.targets, self.sources = alias, targets, sources
```

Comment:
Please don't use this pattern, we removed it from some places in the past (e.g. 4e73d8c04d15f9cbae067249c7ff39dec9d66eb1).

Check out https://github.com/django/django/pull/16805#discussion_r1178697860.